src/applications/static-pages/translation.js

```diff
@@ -10,17 +10,29 @@ const faqI18Content = {
     langToggleLink: '/coronavirus-veteran-frequently-asked-questions-esp',
   },
 };
-
+const configureBreadCrumbLinks = currentLang => {
+  const breadCrumbLinks = document.getElementsByClassName('breadcrumb-link');
+  if (!breadCrumbLinks) {
+    return;
+  }
+  for (const link of breadCrumbLinks) {
+    link.lang = currentLang;
+  }
```

Comment:
Slightly less code?

```suggestion
  breadCrumbLinks.forEach(link => {
    link.lang = currentLang;
  }
```

Reply:
thanks! tried that originally with a run time error b/c `breadCrumbLinks` is an array like object and doesn't have `forEach` on its prototype